api.js

```diff
@@ -5,7 +5,26 @@ module.exports = {
   createTechOps: async function(payload) {
     return axios.post(process.env.API_URL + '/techops', payload, requestHeader())
   },
-  
+
+  createSalesforceCase: async function(payload, authResponse) {
+    return axios.post(
+      process.env.SERVICE_CLOUD_URI + "/api/v1/cases",
+      payload,
+      requestServiceCloudAppHeader(authResponse)
+    );
+  },
+
+  authenticateSalesforce: async function() {
+    let credentials_payload = `{ "username": "${process.env.SERVICE_CLOUD_USER}", "password": "${process.env.SERVICE_CLOUD_PASSWORD}"}`;
```

Comment:
Não está errado da forma que você fez, mas nos casos que queremos transformar um objeto em JSON, podemos usar o [JSON.stringify](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify). Fica aqui só como sugestão:

```suggestion
    const credentialsPayload = JSON.stringify({ username: process.env.SERVICE_CLOUD_USER, password: process.env.SERVICE_CLOUD_PASSWORD });
```

Reply:
E obrigada pelo toque da notação do nome da variável 😬  Estava escrevendo todas no estilo Ruby